spec/models/user_spec.rb

```diff
@@ -1,6 +1,8 @@
-require 'rails_helper'
+# frozen_string_literal: true
+
+require "rails_helper"
 
 RSpec.describe User, type: :model do
-  it { should validate_presence_of(:email) }
-  it { should validate_length_of(:username).is_at_least(1).is_at_most(30) }
+  it { is_expected.to validate_presence_of(:email) }
+  it { is_expected.to validate_length_of(:username).is_at_least(1).is_at_most(30) }
```

Comment:
Run rspec, this test should be failing.
Note: Remember always, before pushing new changes, to run rspec and rubocop.

Reply:
Fixed